### DISCUSSION TITLE
Why am I getting exception when building with Release configuration?

### DISCUSSION THREAD

**Discussion by javidsho** [AvaloniaUI/Avalonia#8463] [Q&A] [answered]
When I build my application with Release config, it throws the following exception upon launching:

```
System.IO.FileNotFoundException: 'Could not load file or assembly 'Avalonia.Controls.DataGrid, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.'
```

I have tried copying out the dll file manually from the debug output as a workaround, but that didn't work either.

**Comment by maxkatz6** [comment, marked as answer]
Add this nuget reference. 
In DEBUG it's transitively referenced through Diagnostic package.

**Reply by javidsho** [reply]
Thanks for the answer :) Somehow I hadn't thought of that!

**Comment by Sfairait** [comment]
@maxkatz6 
Isn't it much simpler to just add in cproj file after:
    `
    <PackageReference Condition="'$(Configuration)' == 'Debug'" Include="Avalonia.Diagnostics" Version="0.10.18" />`
simple line:
	`<PackageReference Condition="'$(Configuration)' == 'Release'" Include="Avalonia.Controls.DataGrid" Version="0.10.18" />`

**Reply by sedwick1024** [reply]
Thanks, that got mine working!  Of course, it's currently v11.2.6...
